storage/remote/otlptranslator/prometheusremotewrite/combined_appender.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2025 The Prometheus Authors
```

Comment:
```suggestion
// Copyright The Prometheus Authors
```

Nit, no need for the date anymore.